README.md

```diff
@@ -14,42 +14,25 @@ All information can be found on [our blog at hello.iRail.be](http://hello.irail.
 
 _note: you'll also need to have [nodejs](https://nodejs.org), [composer](http://getcomposer.org) and PHP curl extension installed on your system_
 
- * Step 1: clone this repo
+ * Step 1: Clone this repo
  * Step 2: `composer install`
- * Step 3: make sure storage is writable: `chmod 777 storage`
+ * Step 3: Make sure storage is writable: `chmod 777 storage`
  * Step 4: Run your test server: `php -S localhost:8008 -t api`
  * Step 5: Enjoy your own iRail API at http://localhost:8008/connections.php?from=Gent%20Sint%20Pieters&to=Antwerp
 
-## Update stations list ##
-
-Stations are updated through the irail/stations composer package. Just perform a `composer update` in the root of the project
-
-## Integrating train occupancy scores ##
-
-We use user feedback and our own knowledge to tell users how busy a train is thank to the crowdfunding campaign of [Spitsgids](https://spitsgids.be).
-
-Optionally, you can thus also set up the occupancy scores on your local machine, but this is a bit more complicated as you will also need mongodb installed:
+**Optional**, if you want to set up the iRail API with occupancy scores you will need to set up a MongoDB database:
 
- * MongoDB: [installation instructions](https://www.mongodb.com/download-center?jmp=nav#community)
- * MongoDB module for PHP, installable through pecl or your systems package manager
+ * Step 6: Install MongoDB: `brew install mongodb`
```

Comment:
brew is Mac OS only... I'd just say: "Make sure you have mongodb installed" with maybe a link to mongodb. Everyone should know on their own how to install this on their machine imo.